distribution/src/config/opensearch.yml

```diff
@@ -99,17 +99,6 @@ ${path.logs}
 #
 # ---------------------------------- Experimental Features -----------------------------------
 #
-# Gates the visibility of the experimental segment replication features until they are production ready.
-#
-#opensearch.experimental.feature.segment_replication_experimental.enabled: false
```

Comment:
Let's not remove the segrep experimental flag as it can still be used for gating experimental changes during 2.11 followups

Reply:
probably we can live with cluster level settings for those.

Reply:
I have left the `SEGMENT_REPLICATION_EXPERIMENTAL` flag as is for now. But it's not being used anywhere for the time being